main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Supposed to be a for loop for 10. Is there a reason why you choose to do a while loop? Also, doesn't it need to be to 10?